test/test_logger.py

```diff
@@ -0,0 +1,122 @@
+import logging
+import os
+import sys
+from subprocess import Popen, PIPE
+from unittest import TestCase, mock
+
+from google.cloud import logging_v2
+
+from bigflow.logger import configure_logging
+
+
+class MockedLoggerHandler(TestCase):
+
+    @mock.patch('bigflow.logger.create_logging_client')
+    def setUp(self, create_logging_client_mock) -> None:
+        create_logging_client_mock.return_value = mock.create_autospec(logging_v2.LoggingServiceV2Client)
+        configure_logging('project-id', 'logger_name', 'workflow-id')
+        self.test_logger = logging.getLogger("logger_name")
+        self.create_logging_client_mock = create_logging_client_mock
+
+
+class LoggerTestCase(MockedLoggerHandler):
+
+    @mock.patch('bigflow.logger.create_logging_client')
+    def test_should_create_correct_logging_link(self, create_logging_client_mock):
+        # given
+        create_logging_client_mock.return_value = mock.create_autospec(logging_v2.LoggingServiceV2Client)
+        with self.assertLogs('another_logger_name', 'INFO') as logs:
+            # when
+            configure_logging('project-id', 'another_logger_name', 'workflow_id')
+
+        # then
+        self.assertEqual(logs.output, ['INFO:another_logger_name:\n'
+ '*************************LOGS LINK************************* \n'
+ ' You can find logs for this workflow here: '
+ 'https://console.cloud.google.com/logs/query;query=logName%3D%22projects%2Fproject-id%2Flogs%2Fanother_logger_name%22%0Alabels.id%3D%22workflow_id%22 \n'
+ '***********************************************************'])
+
+    @mock.patch('bigflow.logger.create_logging_client')
+    def test_should_create_correct_logging_link_without_workflow_id(self, create_logging_client_mock):
+        # given
+        create_logging_client_mock.return_value = mock.create_autospec(logging_v2.LoggingServiceV2Client)
+        with self.assertLogs('another_logger_name', 'INFO') as logs:
+            # when
+            configure_logging('project-id', 'another_logger_name')
+
+        # then
+        self.assertEqual(logs.output, ['INFO:another_logger_name:\n'
+ '*************************LOGS LINK************************* \n'
+ ' You can find logs for this workflow here: '
```

Comment:
Imo souds better:  You can find this workflow logs here:

Reply:
Done